pages/linux/openvpn3.md

```diff
@@ -0,0 +1,24 @@
+# openvpn3
+
+> This is the next generation OpenVPN client for Linux.
```

Comment:
Instead of:
> This is the next generation OpenVPN client for Linux.
Write

What about: 
> "OpenVPN 3 Linux client."

Or:
> The "next generation" OpenVPN client for Linux.

Reply:
updated description